bigframes/ml/linear_model.py

```diff
@@ -160,6 +160,14 @@ def predict(self, X: utils.ArrayType) -> bpd.DataFrame:
 
         return self._bqml_model.predict(X)
 
+    def predict_explain(self, X: utils.ArrayType) -> bpd.DataFrame:
```

Comment:
Looks like there are a few parameters available, too. https://cloud.google.com/bigquery/docs/reference/standard-sql/bigqueryml-syntax-explain-predict

Consider adding support for these. For example, I believe `top_k_features` is relevant for the linear model and is used in the example for it. https://cloud.google.com/bigquery/docs/reference/standard-sql/bigqueryml-syntax-explain-predict#explain_a_prediction_generated_by_a_linear_regression_model

If not, at least add a TODO for these parameters.